### PR TITLE
documentation for graph dumping feature

### PR DESCRIPTION
**Pull request by lewurm**
related mono PR: https://github.com/mono/mono/pull/2769

### CODE REVIEW

File: docs/advanced/runtime/docs/other.md [hunk 1]
```diff
@@ -33,6 +33,70 @@ Displaying JIT debug output
 
 Use the -v -v -v -v command line argument. Set the MONO_VERBOSE_METHOD env variable to display output for only one method.
 
+
+Dumping JIT IR to IGV
+---------------------
+Set `MONO_CFG_DUMP_METHOD` to specify a method to dump over network to a running instance of the [IdealGrpahVisualizer (IGV)](http://ssw.jku.at/General/Staff/TW/igv.html).
+An IGV build that is compatible with the implementation in Mono is available for [Mac](https://www.dropbox.com/s/j08qtuk7ls2q7j8/idealgraphvisualizer_mac_b18b92bc.zip?dl=0) and [Linux/Windows](https://www.dropbox.com/s/3fwsj81df81qrya/idealgraphvisualizer_others_b18b92bc.zip?dl=0) and requires at least JRE 1.7 to run.
```
Comment (alexrp) [inline docs/advanced/runtime/docs/other.md]: Linking to binaries in a Dropbox makes me somewhat uncomfortable. It's not obvious how these builds differ from the official builds of the program, for example.

File: docs/advanced/runtime/docs/other.md [hunk 2]
```diff
@@ -33,6 +33,70 @@ Displaying JIT debug output
 
 Use the -v -v -v -v command line argument. Set the MONO_VERBOSE_METHOD env variable to display output for only one method.
 
+
+Dumping JIT IR to IGV
+---------------------
+Set `MONO_CFG_DUMP_METHOD` to specify a method to dump over network to a running instance of the [IdealGrpahVisualizer (IGV)](http://ssw.jku.at/General/Staff/TW/igv.html).
```
Comment (akoeplinger) [inline docs/advanced/runtime/docs/other.md]: typo: IdealGrpahVisualizer

File: docs/advanced/runtime/docs/other.md (hunk 1 above)
Comment (akoeplinger) [inline docs/advanced/runtime/docs/other.md]: Yeah, that's a no-go. Do we have a fork of the code on github somewhere? You can add a release with the zip's there.

Comment (lewurm) [inline docs/advanced/runtime/docs/other.md]: you're right, providing the builds via Dropbox is a bit sketchy :) The builds from the official page are outdated, IGV is maintained in the Graal repository (`src/share/tools/IdealGraphVisualizer`). I forked the repo and made a release on github: https://github.com/lewurm/GraalJVMCI8/releases/tag/v0.1
I'll update the PR soon.

**Comment by lewurm** [comment]
the mono PR got merged, can we merge this puppy too?

**Comment by akoeplinger** [comment]
Do you not have permissions to merge on the website?

**Comment by lewurm** [comment]
I do, but pressing the merge button is too much responsibility :P

**Comment by akoeplinger** [comment]
Makes sense

gif

**Pull request merged**